api/proto/config/orca.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package proto.config;
+
+option go_package = "github.com/spinnaker/kleat/api/client/config";
+
+import "security/trust_store.proto";
+
+// Configuration for the Orca microservice.
+message Orca {
+  PipelineTemplates pipelineTemplates = 1;
+  WebhookConfig webhook = 2;
+  Defaults default = 3;
+
+  // Configuration for pipeline templates.
+  message PipelineTemplates {
+    // Whether pipeline templates are enabled.
+    bool enabled = 1;
+  }
+
+  // Defaults applicable to the orca microservice.
+  message Defaults {
```

Comment:
Let me know what you think about the nested protocol buffers here.  I know at one point we un-nested because the style guide said that messages should rarely be nested, but in this case the messages are really wrappers that are just defining how orca expects the data so it seemed reasonable to nest?  But I'm definitely interested in your opinion here, I can also see an argument for un-nesting these as `OrcaDefaults` and `OrcaBakeDefaults`.

Reply:
Nesting seems right for these  👍